src/main/java/uk/gov/hmcts/reform/em/stitching/service/DocumentTaskService.java

```diff
@@ -9,6 +9,7 @@
  * Service Interface for managing DocumentTask.
  */
 public interface DocumentTaskService {
+    public static final int CURRENT_VERSION = 1;
```

Comment:
May I suggest that this is incremented as part of the master pipeline / build script. Its only really important when something is released.

Rather than being a flag that may or may not be updated at dev time

Reply:
good plan, I will have a poke around and see what how I might be able to do it. I guess I can use the jenkins build number

Reply:
I think I can use the build number from https://github.com/hmcts/rpa-em-stitching-api/blob/master/src/main/java/uk/gov/hmcts/reform/em/stitching/info/BuildInfo.java

Reply:
sounds sensible